app/presenters/bill-runs/review/base-review.presenter.js

```diff
@@ -196,6 +196,32 @@ function formatIssues(issues) {
   return issues.split(', ')
 }
 
+/**
+ * Format the purposes for a review return
+ *
+ * It is possible that a return log has more than one purpose associated it. If this is the case we need the description
+ * of each purpose from the array of purposes to be concatenated into a single comma separated string ready to present.
+ *
+ * @param {Array} purposes - the purposes taken from the return logs metadata
+ *
+ * @returns {string} the purpose descriptions presented as a comma separated string
+ */
+function formatPurposes(purposes) {
+  const firstPurposeDescription = purposes[0].tertiary.description
```

Comment:
Could this not be done by using array.join(', ')
<img width="233" alt="image" src="https://github.com/user-attachments/assets/e85355d4-78e1-4238-ac2b-41b50280b347" />

Reply:
`function purposes(purposes) {
  return purposes.map((purpose) => {
    return purpose.tertiary.description.join(', ')
  })
}`

Reply:
I cannot perform a `join` on `purpose.tertiary.description` as it's not an array

Reply:
I've modified your idea a bit to get it to work with my array, cheers.